src/main/java/name/valery1707/junit/rule/ConditionalIgnoreRule.java

```diff
@@ -92,11 +104,17 @@ public void evaluate() {
 
 	@Retention(RetentionPolicy.RUNTIME)
 	@Target({ElementType.METHOD})
+	@Repeatable(ConditionalIgnoreList.class)
 	public @interface ConditionalIgnore {
-		//todo Multiply usage with skip if any of rule say to skip
 		Class<? extends IgnoreCondition> condition();
 	}
 
+	@Retention(RetentionPolicy.RUNTIME)
+	@Target({ElementType.METHOD})
```

Comment:
\[Checkstyle\] WARNING: 'annotation def modifier' has incorrect indentation level 8, expected level should be 2\.